client/shared/dev/defaults.bzl

```diff
@@ -0,0 +1,25 @@
+load("@aspect_rules_ts//ts:defs.bzl", _ts_config = "ts_config", _ts_project = "ts_project")
+
+def ts_project(name, composite = True, declaration = True, declaration_map = True, resolve_json_module = True, source_map = True, **kwargs):
+    """Default arguments for ts_project."""
+    _ts_project(
+        name = name,
+        composite = True,
+        declaration = True,
+        declaration_map = True,
+        resolve_json_module = True,
+        source_map = True,
```

Comment:
These properties are set in the root tsconfig.

Reply:
Lets also include a default `tsconfig = "//:tsconfig"` to align with those?